### ISSUE TITLE
Specify strong with css class

### ISSUE THREAD

**Issue by sang-codecademy** [Ollmeister/prj-rev-bwfs-dasmoto#3]
Next time, make sure to specify your strong. For example, do `<strong class = “prices” >`. This makes your code more flexible as it grows in the future and you have to use strong elsewhere. This way you can use it elsewhere if you just specify another class.

https://github.com/Ollmeister/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L16

**Comment by Ollmeister** [comment]
Of course! Wonder why I didn’t think of that. It’s so obvious that this is how it should be done in the first place.
